precompiles/assets-erc20/src/mock.rs

```diff
@@ -229,7 +230,7 @@ impl pallet_assets::Config<ForeignAssetInstance> for Runtime {
 	type Extra = ();
 	type AssetAccountDeposit = AssetAccountDeposit;
 	type WeightInfo = pallet_assets::weights::SubstrateWeight<Runtime>;
-	type RemoveItemsLimit = ConstU32<1000>;
+	type RemoveItemsLimit = ConstU32<656>;
```

Comment:
Magic number, please comment explaining why hehe

Reply:
moved the value to a constant in each runtime and added a comment there